### PR TITLE
Update README with Phoenix live-reload config change

### PR DESCRIPTION
**Pull request by garlandcrow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,21 @@ html lang: "en" do
...
+...
+config :your_app, YourAppWeb.Endpoint,
+  live_reload: [
+    patterns: [
+      # ...
+      # replaces ~r"lib/you_app_web/templates/.*(eex)$"
+      ~r"lib/your_app_web/templates/.*(exs)$"
+    ]
+  ]
+...
```
Comment (mhanberg) [inline README.md]: Can you add this to the same code block as the other configuration, without the `...` and the description (I think it's obvious what it's doing)

File: README.md [hunk 2]
```diff
@@ -153,6 +153,21 @@ html lang: "en" do
...
+```
+
```
Comment (mhanberg) [inline README.md]: Could you also add this to the `Temple.Engine` module doc?

File: README.md [hunk 3]
```diff
@@ -153,6 +153,21 @@ html lang: "en" do
...
 
+Update your config to live-reload the template files.
+```elixir
+# config/dev.exs
+...
+config :your_app, YourAppWeb.Endpoint,
+  live_reload: [
+    patterns: [
+      # ...
+      # replaces ~r"lib/you_app_web/templates/.*(eex)$"
```
Comment (mhanberg) [inline README.md]:
```suggestion
```

**Comment by garlandcrow** [comment]
Updated as requested.

**Pull request merged**